### ISSUE TITLE
Docker image: "Error: Cannot find module '../../package.json'"

### ISSUE THREAD

**Issue by walles** [EXXETA/openapi-cop#401]
Hello!

I can't get the Docker image to work.

Am I doing something wrong here, or is the Docker image broken?

```sh
~ $ docker run -it 'lxlu/openapi-cop:v1.3.2'
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '../../package.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/openapi-cop-docker/src/cli.js:13:20)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
~ [1]$
```

**Comment by LexLuengas** [comment]
Yes, it unfortunately is. Shame our CI did not catch this. I will have a look.
